Source/options.cpp

```diff
@@ -59,6 +59,40 @@ CSimpleIni &GetIni()
 	return ini;
 }
 
+static bool IniChanged = false;
+
+/**
+ * @brief Checks if a ini entry is changed by comparing value before and after
+ */
+struct IniChangedChecker {

```

Comment:
This would usually be a class, as it has private fields. Idiomatically, only things with all-public members are called `struct`.

Reply:
I changed it to `class` 🙂

Reply:
@glebm ping. Are you okay with the changes? 🙂